lighthouse-core/gather/connections/connection.js

```diff
@@ -109,6 +109,11 @@ class Connection {
   }
 
   _handleRawError(object, callback) {
+    // We proactively disable a few domains. Ignore any errors
+    if (object.error.message && object.error.message.includes('DOM agent hasn\'t been enabled')) {
```

Comment:
will this suppress all legitimate errors when the domain isn't enabled?

I wonder if we could instead catch in driver and throw a custom error message so we know it came from our own doing.